app/src/ui/index.tsx

```diff
@@ -176,9 +180,10 @@ const sendErrorWithContext = (
         extra.windowState = currentState.windowState
         extra.accounts = `${currentState.accounts.length}`
 
-        if (__DARWIN__) {
```

Comment:
Forgot to add this comment when I opened the PR: This check should have been removed before. We support this on Windows as of https://github.com/desktop/desktop/pull/9016